ckanext/pages/actions.py

```diff
@@ -14,6 +18,20 @@
 
 
 import db
+
+
+def image_mime_type_validator(file_path):
+    allowed_types = [
+        'image/jpeg',
```

Comment:
We should allow at least '.jpg' too. It would be good to move these values to a config but I wouldn't block a patch for it

Reply:
OK,  thanks